apps/sms/views/shared/js/activity_handler.js

```diff
@@ -380,13 +380,16 @@ var ActivityHandler = {
         }
 
         function continueWithNotification(sender, body) {
-          var title = sender;
+          var titlePromise;
+
           if (Settings.hasSeveralSim() && message.iccId) {
             var simName = Settings.getSimNameByIccId(message.iccId);
-            title = _(
+            titlePromise = formatValue(
```

Comment:
I guess no because the formatValue doc on MDN said:
```
...
formatValue internally waits for all the localization resources to be fetched.  This makes it race condition-proof.
```